File: Python/1038.py
tabela = {1: 4.0, 2: 4.5, 3: 5.0, 4: 2.0, 5: 1.5}
cod, qtde = map(int, input().split())
print("Total: R$ %.2f" % (tabela[cod] * qtde))

File: C/1038.c
#include <stdio.h>

int main(void) {
  int cod;
  int quantidade;
  scanf("%d%d", &cod, &quantidade);
  float valor[5] = {4.00, 4.50, 5.00, 2.00, 1.50};
  printf("Total: R$ %.2f\n", (quantidade*valor[(cod-1)]));
  return 0;
}
